pkg/transport/transport_http.go

```diff
@@ -0,0 +1,107 @@
+package transport
+
+import (
+	"context"
+	"crypto/tls"
+	"errors"
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+// There are other options on http.Transport, but these are the
+// ones which have been configured in gostatsd historically.
+
+const paramHttpDialerKeepAlive = "dialer-keep-alive"
```

Comment:
Was there a reason for not adding a const block instead defining const in front of each?

Reply:
Not that I can tell.  I think I just started with only needing 1 constant and then copy/pasting it a bunch.

Reply:
If we can move this to a const block, it matches the style used throughout the repo already.

Reply:
I'll clean it up in the PR that consumes it.  There's work based off this branch already.